openselery/openselery.py

```diff
@@ -175,99 +193,141 @@ def gather(self):
                                 libIoRepository.github_id)
 
                             # safe project / dependency information
-                            generalProjects.append(gitproject)
-                            generalDependencies.extend(libIoDependencies)
+                            dependencyProjects.append(gitproject)
+
+            for p in dependencyProjects:
+                # grab contributors
+                depContributors = self.githubConnector.grabRemoteProjectContributors(
+                p)
```

Comment:
this newline style of your IDE (or your own weird thing) annoys the hell out of me :D

i mean .. really? 
```
depContributors = self.githubConnector.grabRemoteProjectContributors(
                p)
```

xD